src/js/stores/MesosStateStore.js

```diff
@@ -49,6 +52,20 @@ function assignSchedulerTaskField(task, schedulerTasks) {
   return task;
 }
 
+/**
+ * Assigns a property to task if it belongs to an SDK service.
+ * @param  {Object} task
+ * @param  {Array} service task belongs to
+ * @return {Object} task
+ */
+function flagSDKTask(task, service) {
```

Comment:
Not blocking but it might make sense to move this one along with a `isSDKTask` to `TaskUtil`.